<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>first</class>
 <widget class="QWidget" name="first">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>1600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>421</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab {
    min-width: 140px; /* 选项卡的最小宽度 */
    min-height: 50px; /* 选项卡的最小高度 */
}
QTabBar::tab:selected {
     /* 选中选项卡的背景色 */
	background-color: rgb(4, 212, 167);
    color: black; /* 选中选项卡的文本颜色 */
}
QTabBar::tab
{
    background-color:white; /* 选中选项卡的背景色 */
    color: black; /* 选中选项卡的文本颜色 */
}

</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::South</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabBarAutoHide">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="first">
    <attribute name="title">
     <string>首页</string>
    </attribute>
    <widget class="QStackedWidget" name="changeimage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>401</width>
       <height>261</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QStackedWidget
{
border:none;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="styleSheet">
       <string notr="true">QWidget
{
	
	image: url(:/sourse_images/222.png);
}</string>
      </property>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="styleSheet">
       <string notr="true">QWidget
{

	
	image: url(:/sourse_images/111.png);
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="previousButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border:none;
	image: url(:/sourse_images/right.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="nextButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>190</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border:none;
	image: url(:/sourse_images/left.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>411</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
	background-color: rgb(4, 212, 167);
	color: rgb(255, 255, 255);
border:none;
}</string>
     </property>
     <property name="text">
      <string>    welcome to Easy Park!!!</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="car">
    <attribute name="title">
     <string>车辆</string>
    </attribute>
    <widget class="QPushButton" name="bookbtn">
     <property name="geometry">
      <rect>
       <x>332</x>
       <y>547</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>预约车位</string>
     </property>
    </widget>
    <widget class="QTableView" name="spottab">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>401</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spotnum">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>544</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mine">
    <property name="styleSheet">
     <string notr="true">
QWidget
{
	background-color: rgb(207, 209, 204);
}</string>
    </property>
    <attribute name="title">
     <string>我的</string>
    </attribute>
    <widget class="QPushButton" name="btnwallet">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border:none;
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	text-align: left;
	 padding-left: 20px;
	width: 411px; 
	height: 51px;
}</string>
     </property>
     <property name="text">
      <string>我的钱包</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnbill">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>232</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border:none;
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	text-align: left;
	 padding-left: 20px;
	width: 411px; 
	height: 51px;
}</string>
     </property>
     <property name="text">
      <string>我的账单</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnfault">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>284</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border:none;
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	text-align: left;
	 padding-left: 20px;
	width: 411px; 
	height: 51px;
}</string>
     </property>
     <property name="text">
      <string>故障保修</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnhelp">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>336</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border:none;
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	text-align: left;
	 padding-left: 20px;
	width: 411px; 
	height: 51px;
}</string>
     </property>
     <property name="text">
      <string>帮助中心</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>-1</y>
       <width>401</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget
{
	background-color: rgb(4, 212, 167);
	background-color: rgb(4, 212, 167);
}
QLineEdit
{
	border:none;
	
	color: rgb(255, 255, 255);
	
}</string>
     </property>
     <widget class="QLabel" name="labheadimage">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>90</width>
        <height>90</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	border:none;
		border-radius:45px;
		
	background-color: rgb(239, 255, 228);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="username">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>思源宋体 CN Medium</family>
        <pointsize>18</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="car_plate">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>81</y>
        <width>140</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>车牌</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="phone">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>109</y>
        <width>140</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>电话</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnchange">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border:none;
	image: url(:/sourse_images/left.png);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="email">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>140</y>
        <width>140</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>邮箱</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>440</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border:none;
	border-radius:20px;
	
	background-color: rgb(4, 212, 167);
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>退出登录</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>changeimage</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
